index.bs

```diff
@@ -243,6 +243,50 @@ scenarios. Additional scenarios, including sample code, are given later in [[#sa
     * Web page shows that the selected user is signed in, and navigates to the signed-in page.
 
 
+### New device enrollment ### {#usecase-new-device-enrollment}
+
+This use case scenario illustrates how a [=[RP]=] can leverage a combination of a [=roaming authenticator=] (e.g., a USB security
+key fob) and a [=platform authenticator=] (e.g., a built-in fingerprint sensor) such that the user has:
+
+  - a "primary" [=authenticator=] that they use to authenticate on new-to-them devices (e.g., laptops, desktops) or on such
+    devices that lack a [=platform authenticator=], and
+  - a low-friction means to strongly re-authenticate on devices having [=platform authenticators=].
+
+Note: This approach of registering multiple [=authenticators=] for an account is also useful in account recovery use cases.
+
+Note: The various prompts and user-entered texts in this section are examples, and not normative.
+
+- First, on a desktop computer (lacking a [=platform authenticator=]):
+    * User navigates to `example.com` in a browser and signs in to an existing account using whatever method they have been using
+        (possibly a legacy method such as a password), or creates a new account.
```

Comment:
One more flow:

During new account creation, RP can choose to directly associate a FIDO key with the account without the need to associate an password/other methods with the account. This way, this account can only be unlocked with these FIDO keys and RP's can get the benefit of not having passwords in the first place.

Reply:
I agree this makes sense to point out, but I think putting it into this use case might make it too many things all at once. How about mentioning that in [§1.1.4. Other use cases and configurations][other]?

[other]: https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/884.html#other-configurations

Reply:
@akshayku Bump